components/library/openssl/openssl-1.0.2/engines/pkcs11/e_pk11.c

```diff
@@ -3707,6 +3708,15 @@ hw_aes_instruction_set_present(void)
 #if defined(__amd64) || defined(__i386)
 		present = (ui & AV_386_AES) > 0;
 #elif defined(__sparc)
+/*
+ * These symbols should already be defined from /usr/include/sys/auxv_SPARC.h
+ */
+#ifndef AV_SPARC_AES
+#define AV_SPARC_AES 0
```

Comment:
I run Solaris 11.3 on my desktop.  However, we can't use those values from /usr/include/sys/auxv_SPARC.h because that file is copyright by Oracle.  Setting them to zero has the effect of disabling the hardware AES instructions for CPUs that have them.  That's our best option just now.  Our copy of that header file comes from illumos.  Maybe we could ask the illumos developers to update it.